lib/wallet/txdb.js

```diff
@@ -1360,6 +1393,40 @@ class TXDB {
     });
   }
 
+  /**
+   * Get TX count from the database.
+   * @param {Number?} acct
+   * @returns {Promise} - Returns Number.
+   */
+
+  async getTXCount(acct) {
+    let min, max, parse = null;
+    if (!acct) {
+      min = layout.z.min();
+      max = layout.z.max();
+      parse = (key) => {
+        const [index] = layout.z.decode(key);
+        return index;
+      }
+    } else {
+      assert(typeof acct === 'number');
+      min = layout.x.min(acct);
+      max = layout.x.max(acct);
+      parse = (key) => {
+        const [,index] = layout.x.decode(key);
+        return index;
+      }
+    }
+    const keys = await this.bucket.keys({
+      gte: min,
+      lte: max,
+      limit: 1,
+      reverse: true, // TODO: Verify that `bdb` supports `reverse`
```

Comment:
Note to reviewers: This functionality doesn't yet exist in bdb, my branch locally has the changes needed, there will be a PR to add that to bdb later.

Reply:
There is now a PR for this at: https://github.com/bcoin-org/bdb/pull/4